src/Illuminate/Auth/Console/stubs/make/views/layouts/app.stub

```diff
@@ -15,9 +15,9 @@
 
     <!-- Scripts -->
     <script>
-        window.Laravel = {!! json_encode([
-            'csrfToken' => csrf_token(),
-        ]) !!};
+        window.Laravel = {
+            csrfToken: document.querySelector('meta[name="csrf-token"]').getAttribute('content')
```

Comment:
OK. I see It goes to the session store to fetch the token again. However, I don't think this  approach is more elegant. Why don't we stop using the meta tag and stick to using the javascript variable, instead. What's the point of having the tag anyway? If the goal is to fetch it from Javascript by using DOM manipulation, I think we are better with a global variable. Do you know of any security implications of having the token in a global variable?